hledger/hledger.m4.md

```diff
@@ -4709,15 +4709,15 @@ Values containing `colon:separated:parts` will form a hierarchy, as account name
 Some examples:
 
 ```journal
-2016/02/16 Member Fee Payment
+2016/02/16 Yearly Dues Payment
     assets:bank account                    2 EUR
-    income:member fees                    -2 EUR  ; member: John Doe
+    income:dues                    -2 EUR  ; member: John Doe
```

Comment:
Should we keep the 2's aligned, just for clarity ?

(Procedure: change it locally and "amend" (git commit --amend) the old commit, then force push (git push -f) to this PR branch again.)

Reply:
Sorry for the nit pick, I figure it's good practice! :)